core_api/src/app.py

```diff
@@ -109,12 +134,18 @@ async def create_upload_file(file: UploadFile, ingest: bool = True) -> File:
     Returns:
         File: The file record
     """
+    try:
+        content_type = mime_type_to_content_type[file.content_type]
```

Comment:
Naming is a bit confusing here, you're using the content_type from the file to get the content_type from the mime_type in the dict?